templates/layouts/ec_footer_middle_row.html.twig

```diff
@@ -0,0 +1,8 @@
+<p class="fw-bold pb-2 mb-2">More information on:</p>
```

Comment:
translate static strings, also, we need to import the translations of these static strings, see: 

- https://github.com/openeuropa/oe_theme/tree/3.x/modules/oe_theme_helper/translations
- https://github.com/openeuropa/oe_theme/blob/9ae0e316b23bdb4c95f749145f04906e48ae4e31/modules/oe_theme_helper/oe_theme_helper.info.yml#L16
- https://github.com/openeuropa/oe_theme/blob/9ae0e316b23bdb4c95f749145f04906e48ae4e31/modules/oe_theme_helper/oe_theme_helper.module#L221